src/servicex/servicex.py

```diff
@@ -37,23 +37,31 @@
 import getpass
 
 parser = argparse.ArgumentParser("ServiceX Command Line Interface")
-parser.add_argument("command", choices=["init", "version", "clear"])
+parser.add_argument("command", choices=["init", "clear", "version"])
+parser.add_argument("--namespace", "-n",
+                    default='default',
+                    help="Namespace where secrets should be created")
 parser.add_argument("--cert-dir",
                     default="~/.globus",
                     help="Directory where your grid certs are located")
-parser.add_argument("--namespace", "-n",
-                    default='default',
-                    help="Namespace where secret should be created")
+parser.add_argument("--webhook",
+                    help="Slack incoming webhook URL for notifications about new user registrations")
+
+
+def init_cluster(args):
+    ns = args.namespace
+    create_certs_secret(ns, "grid-certs-secret", args.cert_dir)
+    create_app_secret(ns, "servicex-app-secret", args.webhook)
 
 
-def init_cluster(cert_dir, secret_name, namespace):
+def create_certs_secret(namespace, secret_name, cert_dir):
     passphrase = getpass.getpass("Enter passphrase for certs: ")
 
     try:
         client.CoreV1Api().delete_namespaced_secret(namespace=namespace, name=secret_name)
```

Comment:
Could this code be replaced by a call to `clear_secret`?

Reply:
> It would be better if the command could be used to add the web hook without messing with the certs secret and vice versa.

Good point! I've rewritten things to support subcommands, i.e. we now have `servicex init certs` to only initialize certs, or `servicex clear app` to only clear the secret containing environment variables for the Flask app. To initialize/clear everything, we have `servicex init all`, or simply `servicex init`.